internal/getmodules/get_git.go

```diff
@@ -0,0 +1,45 @@
+package getmodules
+
+import (
+	"net/url"
+	"os/exec"
+
+	"github.com/hashicorp/go-getter"
+)
+
+type GitGetter struct {
+	getter.GitGetter
```

Comment:
Why embed both Getters in the struct? I would give them names so their methods aren't implicitly used/made available by something like `getmodules.GitGetter.SomeMethodInHashiGitGetter()`

```
type GitGetter struct {
	hashiImpl getter.GitGetter
	goImpl GoGitGetter
}
```
or something like that